code/game/jobs/job/science_vr.dm

```diff
@@ -29,4 +29,27 @@
 
 /datum/job/roboticist
 	total_positions = 3
-	pto_type = PTO_SCIENCE
\ No newline at end of file
+	pto_type = PTO_SCIENCE
+
+//////////////////////////////////
+//			Xenobotanist
+//////////////////////////////////
+/datum/job/xenobotanist
```

Comment:
Xenobotanist has no pto_type defined as far as I can see.

Reply:
I needa add that.

Reply:
This has been fixed.